docs/course_materials/variables_data_types.html

```diff
@@ -463,14 +472,7 @@ <h3><b>Manipulations</b></h3>
           s.strip() # -> 'hello!'
                     </code></pre>
                 </section>
-                <section>
-                    <h3><b>+ and *</b></h3>
```

Comment:
why did you remove this one?
consider moving it to the extras instead.

Reply:
I didn't remove it, just change the location,